pkg/bootkube/create.go

```diff
@@ -164,23 +170,28 @@ func (c *creater) createManifests(manifests []manifest) (ok bool) {
 		}
 	}
 
-	create := func(m manifest) {
+	create := func(m manifest) error {
 		if err := c.create(m); err != nil {
 			ok = false
-			UserOutput("Failed creating %s: %v", m, err)
-		} else {
-			UserOutput("Created %s\n", m)
+			UserOutput("Failed creating %s: %v\n", m, err)
+			return err
 		}
+		UserOutput("Created %s\n", m)
+		return nil
 	}
 
 	// Create all namespaces first
 	for _, m := range namespaces {
-		create(m)
+		if err := create(m); err != nil && c.strict {
+			return
```

Comment:
Do you need to set ok=false here?

Or better, I think we want to explicitly `return false` here, right?

Reply:
thanks. fixed it to explicitly return false